scripts/publish-metadata.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import sys
+import requests
+import json
+import subprocess
+
+DESCRIPTION = """
```

Comment:
I would also explain that the `requests` library is necessary, and that `gcloud` needs to be installed.